templates/editCounter/timecard.html.twig

```diff
@@ -122,21 +132,28 @@
                         beginAtZero: true,
                         min: 0,
                         max: 23,
-                        stepSize: 2,
+                        stepSize: 1,
                         reverse: i18nRTL,
+                        padding: 0,
                         callback: function(value) {
                             if (value % 2 === 0) {
                                 return value + ":00";
+                            } else {
+                                return '';
                             }
                         }
                     }
                 }]
             },
             tooltips: {
+                displayColors: false,
                 callbacks: {
+                    title: function(items, data) {
+                        return days[7 - items[0].yLabel + 1] + ' ' + items[0].xLabel + ':00';
+                    },
                     label: function(item) {
-                        // yLabels are reversed, and zero-indexed
-                        return days[7 - item.yLabel + 1] + ' ' + item.xLabel + ':00';
+                        var numEdits = [timeCardDatasets[item.datasetIndex].data[item.index].value];
+                        return`${numEdits} ${$.i18n('num-edits', [numEdits])}`;
```

Comment:
May I ask why is there hard to translate string interpolation instead of reusing the parameter inside a message like `$1 {{PLURAL:$1|edit|edits}}`? 🤔

Reply:
In the Twig templates, we format the first number (e.g. add commas), which wouldn't allow us to use it in PLURAL. JS doesn't have the same formatting. Anyway, I know we could just have $2 for the formatted number, and $1 for use in PLURAL, but it's been this way since the beginning, so I just followed the same pattern https://github.com/x-tools/xtools/blob/e2569fa563758b795fc9e7ba0ae472de410086f1/i18n/en.json#L264-L287